# GetBrightness.srv
---
float32 level
